### ISSUE TITLE
New feature: add/remove git remotes

### ISSUE THREAD

**Issue by hsablonniere** [isomorphic-git/isomorphic-git#344]
Hey,

I would like a new command to add a new git remote.

1. I guess this means we would also need a remove command?
2. Does this mean we also need an edit command?
3. It seems that this only requires to edit the local config file, isn't it?

WDYT?

**Comment by wmhilton** [comment]
> 2. Does this mean we also need an edit command?

maaaaaaybe? Does anyone ever do this though? Maybe more useful would be a listRemotes command

It would be pretty simple right? basically

```js
git.remote({fs, gitdir, remote: 'name', url })
git.listRemotes({fs, gitdir})
git.deleteRemote({fs, gitdir, remote: 'name'})
```

and yes, it just edits the config file. I read through the source code of git to determine this, but "list remotes" works by reading the config file and listing the remotes configured there.

**Comment by wmhilton** [comment]
Ah, I see list remotes is covered by #345

**Comment by hsablonniere** [comment]
I would rename `git.remote` to `git.addRemote` for consistency maybe.

WDYT?

**Comment by hsablonniere** [comment]
I think the edit is not needed right now.
It a very rare case and people can do a config if they need to.

**Comment by hsablonniere** [comment]
Fixed in #361 :+1:
